src/atoms/breadcrumbs/Breadcrumbs.spec.js

```diff
@@ -4,14 +4,19 @@ import ABreadcrumbs from './Breadcrumbs.vue'
 
 describe('Breadcrumbs', () => {
   it('has default structure', () => {
```

Comment:
Maybe we should check if the last element is clickable?

Reply:
Imo is not needed right now, we are depend on `$breadcrumbs__link-pointer-events--current` which allows us to change it dynamically so I am not sure if we can check it on js level